sql/sys_vars.cc

```diff
@@ -5217,6 +5217,13 @@ static Sys_var_uint Sys_repl_report_port(
        "to the slave. If not sure, leave this option unset",
        READ_ONLY GLOBAL_VAR(report_port), CMD_LINE(REQUIRED_ARG),
        VALID_RANGE(0, UINT_MAX), DEFAULT(0), BLOCK_SIZE(1));
+
+static Sys_var_on_access<Sys_var_mybool,
+       PRIV_STMT_SHOW_SLAVE_HOSTS, PRIV_STMT_SHOW_SLAVE_HOSTS>
+Sys_show_slave_auth_info(
+       "show_slave_auth_info",
+       "Show user and password in SHOW SLAVE HOSTS on this master",
+       SESSION_VAR(show_slave_auth_info), CMD_LINE(OPT_ARG), DEFAULT(FALSE));
```

Comment:
How valuable is `@@GLOBAL.show_slave_auth_info= ON`?

Reply:
Well, you may have already found this out, but it doesn't use the values of the connecting replication user, but rather the values of the options `report_user` and `report_password` configured on the slave. As to its value, I have never actually seen anyone use it. I did some Googling to see if people use it or "report_user/password", and have turned up nothing. Then I also checked the commit history and also couldn't find out why it was added in the first place. So I see your point.. Though investigating it would be a different issue, and for the overall agenda of MDEV-28672 we still need it visible, despite the fact that likely no one uses it..